.env-template

```diff
@@ -16,4 +16,6 @@ VUE_APP_PINATA_REQUIRED_DOCUMENT=
 VUE_APP_OCTOPUS_EXPLORER=
 VUE_APP_SERVICE_REQUEST_LINK=
 VUE_APP_PHYSICAL_HEALTH_TIMELINE_ID=
-VUE_APP_MENTAL_HEALTH_TIMELINE_ID=
\ No newline at end of file
+VUE_APP_MENTAL_HEALTH_TIMELINE_ID=
+VUE_APP_DEBIO_ADMIN_JWT=
+VUE_APP_MYRIAD=
```

Comment:
change to `VUE_APP_MYRIAD_URL`

Reply:
it has been changed now , however I still got error when use client's jwt. I have put the ss on https://blocksphere2020.atlassian.net/browse/DBIO-3247